src/components/PipelineRun/PipelineRunDetailsView/tabs/PipelineRunDetailsTab.tsx

```diff
@@ -122,6 +122,10 @@ const PipelineRunDetailsTab: React.FC = () => {
     namespace;
 
   const showSbom = sboms && ((sboms.length === 1 && sboms[0].url) || sboms.length > 1);
+  const showFailedMessage =
```

Comment:
conditionsRunStatus in src/utils/pipeline-utils.ts have some similar status check, could you help to check whether we are able to reuse some directly? 
or why we cannot resue pipelineStatus == Failed directly?